File: module_login/src/main/java/debug/LoginApplication.java
package debug;

import com.ebook.api.RetrofitManager;
import com.ebook.common.BaseApplication;
import com.ebook.login.interceptor.LoginInterceptor;
import com.therouter.router.NavigatorKt;

public class LoginApplication extends BaseApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        RetrofitManager.init(this);
        NavigatorKt.addRouterReplaceInterceptor(new LoginInterceptor());
    }
}

File: module_login/src/main/test/debug/LoginApplication.kt
package debug

import com.ebook.common.BaseApplication
import com.ebook.common.interceptor.LoginInterceptor
import com.therouter.router.addRouterReplaceInterceptor
import dagger.hilt.android.HiltAndroidApp

@HiltAndroidApp
class LoginApplication : BaseApplication() {
    override fun onCreate() {
        super.onCreate()
        addRouterReplaceInterceptor(LoginInterceptor())
    }
}
